### PR TITLE
fix: EDV panic - index out of range

### LINKED ISSUE #202: EDV panic - index out of range
```
2021/03/07 20:14:30 http: panic serving 10.79.0.10:60704: runtime error: index out of range [2] with length 2
goroutine 11545 [running]:
net/http.(*conn).serve.func1(0xc00034b720)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc0006c86e0)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc000033340, 0xc0001a0c00)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc000033340, 0xc0001a0c00)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc00034b720, 0xedc720, 0xc0000d5b40)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:14:44 http: panic serving 10.79.0.10:61234: runtime error: index out of range [2] with length 2
goroutine 11552 [running]:
net/http.(*conn).serve.func1(0xc00034b9a0)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc000308720)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc0000af5e0, 0xc000128b00)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc0000af5e0, 0xc000128b00)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc00034b9a0, 0xedc720, 0xc000437b80)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:15:04 http: panic serving 10.79.0.10:62296: runtime error: index out of range [2] with length 2
goroutine 11577 [running]:
net/http.(*conn).serve.func1(0xc00034a820)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc0006c80e0)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc000032380, 0xc0000c7200)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc000032380, 0xc0000c7200)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc00034a820, 0xedc720, 0xc0006ca5c0)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:15:18 http: panic serving 172.16.0.1:53882: runtime error: index out of range [2] with length 2
goroutine 11588 [running]:
net/http.(*conn).serve.func1(0xc0004845a0)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc0006c82c0)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc0000328c0, 0xc0003b0600)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc0000328c0, 0xc0003b0600)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc0004845a0, 0xedc720, 0xc0006caf00)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:15:23 http: panic serving 172.16.0.1:53884: runtime error: index out of range [2] with length 2
goroutine 11607 [running]:
net/http.(*conn).serve.func1(0xc00034adc0)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc0006c83a0)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc000032a80, 0xc0003b0c00)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc000032a80, 0xc0003b0c00)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc00034adc0, 0xedc720, 0xc0006cb100)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:16:23 http: panic serving 10.79.1.96:62568: runtime error: index out of range [2] with length 2
goroutine 11656 [running]:
net/http.(*conn).serve.func1(0xc000485180)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc0006c87a0)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc0000335e0, 0xc0001b1600)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc0000335e0, 0xc0001b1600)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc000485180, 0xedc720, 0xc0006cbec0)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:16:23 http: panic serving 10.79.1.96:62570: runtime error: index out of range [2] with length 2
goroutine 11657 [running]:
net/http.(*conn).serve.func1(0xc000485220)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc000308720)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc0000af500, 0xc00017a700)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc0000af500, 0xc00017a700)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc000485220, 0xedc720, 0xc000153c00)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:16:34 http: panic serving 10.79.1.96:63268: runtime error: index out of range [2] with length 2
goroutine 11665 [running]:
net/http.(*conn).serve.func1(0xc0004854a0)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc0006c88c0)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc000033880, 0xc00017b500)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc000033880, 0xc00017b500)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc0004854a0, 0xedc720, 0xc0000d43c0)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:16:38 http: panic serving 10.79.1.96:63356: runtime error: index out of range [2] with length 2
goroutine 11685 [running]:
net/http.(*conn).serve.func1(0xc0004855e0)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc0003089e0)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc0000afa40, 0xc0001ce100)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc0000afa40, 0xc0001ce100)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc0004855e0, 0xedc720, 0xc0003aa6c0)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:16:38 http: panic serving 10.79.1.96:63362: runtime error: index out of range [2] with length 2
goroutine 11671 [running]:
net/http.(*conn).serve.func1(0xc00034b9a0)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc0006c89e0)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc000033a40, 0xc0001ce200)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc000033a40, 0xc0001ce200)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc00034b9a0, 0xedc720, 0xc0003ab080)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:16:43 http: panic serving 10.79.0.10:3580: runtime error: index out of range [2] with length 2
goroutine 11675 [running]:
net/http.(*conn).serve.func1(0xc00034bae0)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc0006c8a80)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc000033c00, 0xc0001ce600)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc000033c00, 0xc0001ce600)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc00034bae0, 0xedc720, 0xc0003ab540)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:16:45 http: panic serving 10.79.1.96:62566: runtime error: index out of range [2] with length 2
goroutine 11646 [running]:
net/http.(*conn).serve.func1(0xc00034b7c0)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc000308ac0)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc0000afce0, 0xc0001b1500)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc0000afce0, 0xc0001b1500)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc00034b7c0, 0xedc720, 0xc0006cbe00)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:16:56 http: panic serving 10.79.1.96:64420: runtime error: index out of range [2] with length 2
goroutine 11703 [running]:
net/http.(*conn).serve.func1(0xc00034a820)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc0006c8120)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc000032380, 0xc0001ce600)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc000032380, 0xc0001ce600)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc00034a820, 0xedc720, 0xc0003aa540)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:16:56 http: panic serving 10.79.1.96:64422: runtime error: index out of range [2] with length 2
goroutine 11692 [running]:
net/http.(*conn).serve.func1(0xc000484000)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc0006c8240)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc000032460, 0xc0001ce800)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc000032460, 0xc0001ce800)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc000484000, 0xedc720, 0xc0003aad40)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:17:04 http: panic serving 10.79.1.96:64862: runtime error: index out of range [2] with length 2
goroutine 11695 [running]:
net/http.(*conn).serve.func1(0xc0004843c0)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc0003081a0)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc0000ae540, 0xc00017a800)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc0000ae540, 0xc00017a800)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc0004843c0, 0xedc720, 0xc0000d48c0)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:17:11 http: panic serving 10.79.1.96:1242: runtime error: index out of range [2] with length 2
goroutine 11718 [running]:
net/http.(*conn).serve.func1(0xc000484640)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc0006c8480)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc0000328c0, 0xc0000c6200)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc0000328c0, 0xc0000c6200)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc000484640, 0xedc720, 0xc0003ab980)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
2021/03/07 20:17:52 http: panic serving 172.16.0.1:53990: runtime error: index out of range [2] with length 2
goroutine 11749 [running]:
net/http.(*conn).serve.func1(0xc000484b40)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0xc61e20, 0xc000308500)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/trustbloc/edv/cmd/edv-rest/startcmd.(*httpHandler).ServeHTTP(0xc00000f720, 0xeda420, 0xc0000aee00, 0xc0001a0d00)
	/go/src/github.com/trustbloc/edv/cmd/edv-rest/startcmd/start.go:885 +0x43a
net/http.serverHandler.ServeHTTP(0xc0000320e0, 0xeda420, 0xc0000aee00, 0xc0001a0d00)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc000484b40, 0xedc720, 0xc0006ca100)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
```

### PR DESCRIPTION
**Pull request by fqutishat**
closes #202

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

**Review by DRK3 — APPROVED** [review]

**Comment by troyronda** [comment]
@clabot check

**Comment by troyronda** [comment]
@cla-bot check

File: cmd/edv-rest/startcmd/start.go [hunk 1]
```diff
@@ -874,14 +874,14 @@ func (h *httpHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if r.RequestURI == createVaultPath || r.RequestURI == healthCheckPath {
+	s := strings.SplitAfter(r.RequestURI, "/")
+
+	if len(s) < 3 || r.RequestURI == createVaultPath || r.RequestURI == healthCheckPath {
```
Comment (DRK3) [inline cmd/edv-rest/startcmd/start.go:879]: Do you need to check for all three conditions, or would `len(s) < 3` suffice?

**Review by rolsonquadras — APPROVED** [review]

File: cmd/edv-rest/startcmd/start.go (hunk 1 above)
Comment (fqutishat) [reply cmd/edv-rest/startcmd/start.go]: correct. I change it

**Pull request merged**